modules/book.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export */
+// Class book, to define a book
```

Comment:
- [OPTIONAL ]Kindly remove unused code in your project for your codes to look professional and for good practice :wink: